packages/@secretlint/secretlint-rule-npm/src/index.ts

```diff
@@ -123,6 +168,7 @@ export const creator: SecretLintRuleCreator<Options> = {
                 } else if (isNpmrc(source.filePath)) {
                     reportIfFound_AuthTokenInNpmrc({ source, options: normalizedOptions, context, t });
                 }
+                reportIfFound_NPM_ACCESS_TOKEN({ source, options: normalizedOptions, context, t });
```

Comment:
It always check npm access token in texts

Reply:
[
![Uploading IMG_20210915_173703.jpg…]()
](url)

Reply:
Ochrona prywatności